.github/workflows/automatic-doc-checks.yml

```diff
@@ -0,0 +1,16 @@
+name: Main Documentation Checks
+
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
```

Comment:
This is a little bit strange, that we are running CI only on PR, but this will be triggered always.